lib/ecto/adapters/myxql/connection.ex

```diff
@@ -207,6 +207,60 @@ if Code.ensure_loaded?(MyXQL) do
       ["DELETE FROM ", quote_table(prefix, table), " WHERE " | filters]
     end
 
+    @impl true
+    # Opts are deprecated, see Notes at https://dev.mysql.com/doc/refman/5.7/en/explain.html
+    def explain_query(query, _opts \\ []) do
+      {
+        ["EXPLAIN ", query],
+        fn result ->
+          format_result_as_table(result.columns, result.rows) |> IO.iodata_to_binary()
+        end
+      }
+    end
+
+    defp format_result_as_table(columns, rows) do
```

Comment:
@josevalim decided to go with a tabular format like @fxn did on Rails because that's the default output and it's more readable and probably more popular. Note the code is inspired by Rails implementation. Btw thanks again @fxn :)